samples/python/topology/simple/transform_sample_functions.py

```diff
@@ -1,11 +1,33 @@
 def int_strings_transform():
-    return ["325", "457", "9325"]
-    
+    """
+    Returns an iterable of strings
+    """
+    return ["325", "457", "9325"]
+    
 def string_to_int(t):
-    return int(t)
-
+    """
+    Converts a string to integer
+    Args:
+        t (string): tuple
+    Returns:
+        int        
+    """
+    return int(t)
+
 class AddNum:
+    """
+    A callable class that performs addition
+    Args:
+        increment (int): number to add
```

Comment:
Minor point that nothing in this class is specific to int, that's the beauty of Python.